castervoice/lib/ctrl/mgr/rule_details.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, name=None, function_context=None, executable=None, title=None
     def _calculate_filepath_from_frame(stack, index):
         try:
             frame = stack[index]
-            module = inspect.getmodule(frame[0])
-            filepath = module.__file__.replace("\\", "/")  
+            # module = inspect.getmodule(frame[1])
+            filepath = frame[1].replace("\\", "/")
```

Comment:
Would it be better to let Path lib handle file path translation?

Reply:
It would. :)